tic-tac-toe-client/src/main/java/Constants.java

```diff
@@ -0,0 +1,22 @@
+public interface Constants {
```

Comment:
I see here code (which is for task#2) and tests (which is for task#3)
So
1) what task is it for?
2) if it is for task#2 then what code will be run on HW?

Reply:
1. There are tasks 2 and 3 in latest 4 commits in this branch.
2. main.c file in folder "tic-tac-toe-server" is code to be run on HW

Reply:
in futer please create separate PR for seprate tasks

Reply:
Okay, I got it. Sorry